packages/dtslint/src/lint.ts

```diff
@@ -22,6 +22,7 @@ export async function lint(
   const estrees = [
     tryResolve("@typescript-eslint/typescript-estree"),
     tryResolve("@typescript-eslint/typescript-estree", { paths: [dirPath] }),
+    tryResolve("@typescript-eslint/typescript-estree", { paths: [resolve(__dirname, "../../eslint-plugin")] }),
```

Comment:
I'm super surprised by this; when I install things, the previous two are the only ones that exist on disk. Is it the case that things got weird when I bumped deps and left your node_modules in a mixed state?

(Not against this line, though I am suspicious of the relative path encoding something about node_modules' layout)